firewood/src/merkle/node.rs

```diff
@@ -8,6 +8,7 @@ use crate::{
 use bincode::{Error, Options};
 use bitflags::bitflags;
 use enum_as_inner::EnumAsInner;
+use log::debug;
```

Comment:
You should pull this out into an internal module where you do the conditional compilation. There, you can export your own debug macro that will just be a `() => {}` (not a no-op... but a no-code? haha) if the flag isn't set. 

That way you can get rid of the feature flag and won't forget about it. You can do the same thing with the other log-levels as well.

Reply:
Fixed in latest